packages/data/src/lib/data-view/data-view.ts

```diff
@@ -0,0 +1,132 @@
+import { CartoError, WithEvents } from '@carto/toolkit-core';
+import { Layer, CARTOSource } from '@carto/toolkit-viz';
+import { groupValuesByAnotherColumn } from '../aggregations/grouping';
+import {
+  applyAggregations,
+  AggregationTypes
+} from '../aggregations/aggregations';
+
+export class DataView extends WithEvents {
+  private source: CARTOSource | Layer;
```

Comment:
Thinking about 'source' name... Should we change it to something that accommodates better the 'Layer' option? Maybe **origin** ? So to avoid using the word 'source' that might be associated with just a CARTOSource